data/src/main/java/com/kdjj/data/di/RepositoryModule.kt

```diff
@@ -0,0 +1,14 @@
+package com.kdjj.data.di
+
+import com.kdjj.data.repository.RecipeListRepositoryImpl
+import com.kdjj.domain.repository.RecipeListRepository
+import dagger.Binds
+import dagger.Module
+import javax.inject.Singleton
+
+@Module
+abstract class RepositoryModule {
+    @Binds
```

Comment:
한줄 띄어주시는게 가독성이 더 좋을 것 같아요!
```suggestion
abstract class RepositoryModule {

    @Binds
```

Reply:
추가했습니다 !